server/audit_math/ballot_polling.py

```diff
@@ -3,17 +3,11 @@
 ballot polling audits.
 """
 from typing import Dict, Tuple, Optional
-from typing_extensions import Literal, TypedDict
+
 from ..models import AuditMathType
 from .sampler_contest import Contest
 from . import bravo, minerva
-
-
-class SampleSizeOption(TypedDict):
-    type: Optional[Literal["ASN"]]
-    size: int
-    prob: Optional[float]
-
+from .ballot_polling_types import SampleSizeOption
```

Comment:
is there a reason this got split into it's own file? (Not complaining, just curious if it's a stylistic choice or a type-checker satisfying choice)

Reply:
Good q! The motivation was avoiding a circular dependency. If I'd kept `SampleSizeOption` in `server/audit_math/ballot_polling.py`, `server/audit_math/bravo.py` would have to import from `server/audit_math/ballot_polling.py`, but `server/audit_math/ballot_polling.py` already imports from `server/audit_math/bravo.py` 🔁